cli/src/list_devices.rs

```diff
@@ -1,23 +1,20 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
-use clap::Args;
-use std::path::PathBuf;
-
-use libparsec::{get_default_config_dir, list_available_devices};
+use libparsec::list_available_devices;
 
 use crate::utils::*;
 
-#[derive(Args)]
+#[derive(clap::Parser)]
 pub struct ListDevices {
-    /// Parsec config directory
-    #[arg(short, long, default_value_os_t = get_default_config_dir())]
-    config_dir: PathBuf,
+    #[clap(flatten)]
+    config: ConfigSharedOpts,
 }
 
 pub async fn list_devices(list_devices: ListDevices) -> anyhow::Result<()> {
-    let config_dir = list_devices.config_dir;
-    let config_dir_str = config_dir.to_string_lossy();
+    let config_dir = list_devices.config.config_dir;
+    log::trace!("Listing devices under {}", config_dir.display());
```

Comment:
why not use the same syntax that other messages:
```suggestion
    log::trace!("Listing devices (config_dir={})", config_dir.display());
```